.cypress/utils/constants.js

```diff
@@ -10,7 +10,8 @@ export const COMMAND_TIMEOUT_LONG = 10000;
 export const TRACE_ID = '8832ed6abbb2a83516461960c89af49d';
 export const SPAN_ID = 'a673bc074b438374';
 export const SERVICE_NAME = 'frontend-client';
-export const SERVICE_SPAN_ID = '7df5609a6d104736';
+export const SERVICE_SPAN_ID = 'e275ac9d21929e9b';
+export const AUTH_SERVICE_SPAN_ID = '277a5934acf55dcf';
```

Comment:
What are these constants for/why change them?

Reply:
Missed one of these during clean up - but the other one is to make a search/find a specific span instead of assuming Opensearch is returning them deterministically